packages/diagrams/lib/src/expanded.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+const String _column = 'expanded_column';
+const String _row = 'expanded_row';
+
+class ExpandedDiagram extends StatelessWidget implements DiagramMetadata {
+  const ExpandedDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget returnWidget;
+
+    switch (name) {
+      case _column:
```

Comment:
The column appears to be filling the entire viewport, while the row has a fixed height of 100. Let's make the column a fixed width of 100 to match.
Also, let's make the expanded/not expanded colors the same between examples to better illustrate which is which.

Reply:
Oh my. For some reason I thought the word "Expanded" would run longer than 100 pixels wide, but that wasn't the case -- yet I made these changes anyway to make sure it fit and the column didn't blend with the app bar! Egg on my face, and 100 pixel width + colors returned to their natural state.